discord/voice_client.py

```diff
@@ -525,12 +345,22 @@ async def move_to(self, channel: Optional[abc.Snowflake]) -> None:
         -----------
         channel: Optional[:class:`abc.Snowflake`]
             The channel to move to. Must be a voice channel.
+        timeout: Optional[:class:`float`]
+            How long to wait for the move to complete.
+
+            .. versionchanged:: 2.4
```

Comment:
This needs a description

Reply:
This was supposed to be changed to a `versionadded` when I moved it under the parameter section but I forgot.